snapcraft/internal/pluginhandler/__init__.py

```diff
@@ -564,6 +564,9 @@ def prime(self, force=False) -> None:
         linker_compatible = (e.is_linker_compatible(linker='ld-2.23.so')
                              for e in elf_files)
         if not all((x for x in linker_compatible)):
+            if 'libc6' not in self.stage_packages:
+                raise errors.StagePackageMissingError(package='libc6')
```

Comment:
Okay so we have an error here, but the patching text above is just a warning.... there's a disconnect here.

Reply:
When you say above, I hope you mean above in this review screen. Then yes, I will take care of that.